recipe/meta.yaml

```diff
@@ -10,8 +10,6 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256sum }}
-  patches:
-    - patches/0001-Fix-integer-check.patch
 
 build:
   number: 0
```

Comment:
You can add a new line

`    skip: True  # [win]`

To skip the windows builds for this release until the setup issue is fixed.